src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,17 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<String> collect;
```

Comment:
If I understand correctly, we need to use TreeSet here. Set is an interface like a List, but we need to use TreeSet which is one of the Set implementations.

Reply:
done